cinq_collector_aws/account.py

```diff
@@ -98,7 +79,7 @@ def update_s3buckets(self):
                     else:
                         self.log.error('There was a problem collecting bucket policy for bucket {} on account {}, {}'
                                        .format(data.name, self.account, e.response))
-                        bucket_policy = 'Unavailable'
+                        bucket_policy = 'cinq cant poll'
```

Comment:
Fix punctuation on "cant"

`bucket_policy = 'cinq can\'t poll'`
or
`bucket_policy = "cinq can't poll"`

```suggestion
                        bucket_policy = "cinq can't poll"
```